Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -39,7 +41,16 @@ - (instancetype)initWithTox:(OCTTox *)tox
     _audioEngine = [OCTAudioEngine new];
     _audioEngine.toxav = self.toxAV;
 
+    OCTConverterFriend *friendConverter = [OCTConverterFriend new];
+    friendConverter.dataSource = self;
+
+    OCTConverterMessage *messageConverter = [OCTConverterMessage new];
+    messageConverter.converterFriend = friendConverter;
+
     _chatConverter = [OCTConverterChat new];
+    _chatConverter.delegate = self;
+    _chatConverter.converterFriend = friendConverter;
+    _chatConverter.converterMessage = messageConverter;
```

Comment:
Also you have to set `converterFriend` property of `messageConverter`.

Actually this all converter logic is to messy. I'm gonna refactor them in a few days.